### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Daanra**
Fixes:
https://github.com/Daanra/net-computing/issues/35
https://github.com/Daanra/net-computing/issues/28
https://github.com/Daanra/net-computing/issues/25
https://github.com/Daanra/net-computing/issues/24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +45,22 @@ To login into the webinterface, you can use the following credentials:
...
+
+
+To run a client or a regional server, you might first have to configure the proper IP settings in their corresponding config files. 
+To run a regional server the following command can be used:
+
+`python3 app/regionalserver/regionalServer.py`
+
+To run a client the following command can be used:
+
+`python3 app/client/client.py`
```
Comment (milton19) [inline README.md]: python3 app/client/client.py bikeId

File: app/database.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    db.session.add(admin)
+    db.session.add(guest)
+    db.session.commit()
+
+
+def add_bikes():
+    db.session.add(Bike(1, 53.2053186, 6.5519019, datetime.datetime.now(), "My black bike"))
+    db.session.add(Bike(2, 53.2193264, 6.5575646, datetime.datetime.now(), "Purple Turbo Motor bicycle"))
+    db.session.add(Bike(3, 53.2242074, 6.5510306, datetime.datetime.now(), "Mom's bike"))
+    db.session.add(Bike(1, 53.2195224, 6.5663996, datetime.datetime.now(), "Super Flying bicycle"))
```
Comment (leonardas103) [inline app/database.py]: Bike id 1 used again?

put all the bikes in the city center
(1, 53.218703, 6.567847)
(2, 53.219235, 6.563288)
(3, 53.216899, 6.563844)
(4, 53.218360, 6.568546)

File: app/regionalserver/regionalServer.py [hunk 3]
```diff
@@ -22,22 +22,24 @@ def __init__(self, conn, addr, port):
...
         while True:
             data = self.socket.recv(1024)
             if not data:
                 self.socket.close()
                 break
-            logger.info("[%s] RegionalServer got: %s" % (threadId, data.decode()))
-            if checkIntregity(data.decode()):
+            logger.info("[%s] RegionalServer got: %s" % (thread_id, data.decode()))
+            if check_integrity(data.decode()):
                 channel = sender.createChannel(mainServerHost)
```
Comment (leonardas103) [inline app/regionalserver/regionalServer.py]: should be:
sender.create_channel

File: app/database.py (hunk 2 above)
Comment (Daanra) [reply app/database.py]: That is not the bike id, it is the owner id. Account with id 1 has two bicycles

**Pull request merged**